1_space_odyssey/js/olav-9000/script.js

```diff
@@ -6,29 +6,35 @@
     c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
 */
 
-var isAlive = true;
-var name = "Hoff-9000";
-var hasJetPack = true;
-var suitColor = "neongreen";
-var eyes = true;
-var eyeColor = "blue";
-var hairColor = "blonde";
-var legs = 2;
-var arms = 2;
-var hands = 2;
-var ears = 2;
-var canWalk = true;
-var canRun = false;
-var canFly = true;
-var hasWeapon = true;
-var isInGoodMood = true;
-
-var voiceover = "David Hasslehoff";
-var bodytype = "car";
-var evilAI = "EvilArcade";
-var evilAIBodyType = "Packman machine";
-
-
+//refactor robot variables
+var hoff9000 = {
```

Comment:
cool cool cool yes!

you can take it a step further 
` - Your android needs a head, upperBody, lowerBody & [ anyThingElseYouCanDreamOf ].`

```js
var obj = {
     head: {
        ...
     },
     upperBody: {
        ...
     },
     lowerBody: {
        ...
     },
}
```